### ISSUE TITLE
Delayed audio output

### ISSUE THREAD

**Issue by PavanLuca** [flathub/com.adobe.Flash-Player-Projector#66]
Audio output is in delay compared to events. Personally I noticed this with Thing Thing Arena but surely happens with any, the Windows version hasn't this problem but FPS is evidently lower. Here below there are my system info.
![Informazioni sul sistema](https://github.com/flathub/com.adobe.Flash-Player-Projector/assets/123581658/d078cbc9-bbb9-4984-83f8-312c4af3d0a7)

**Comment by TheHENOOB** [comment]
On my end, I am also experiencing Audio Delay.

In this case with Canabalt:
https://www.newgrounds.com/portal/view/510303
https://adamatomic.com/canabalt/

**Comment by PavanLuca** [comment]
Do you have PulseAudio or another audio source?

**Comment by TheHENOOB** [comment]
> Do you have PulseAudio or another audio source?

I am not so sure, but I think Linux Mint uses PulseAudio as the default audio source.

**Comment by PavanLuca** [comment]
Yes, you can also see it on your system monitor software on processes section, I see `pulseaudio` on mine.

**Comment by PavanLuca** [comment]
Yes, you can also see it on your system monitor software on processes section, I see pulseaudio on mine. It's possible that the problem is limited to who has PulseAudio.

**Comment by PavanLuca** [comment]
Yes, you can also see it on your system monitor software on processes section, I see `pulseaudio` on mine. It's possible that the problem is limited to who has PulseAudio.

**Comment by jedenastka** [comment]
I have the same issue with PipeWire 1.0.5, with no PulseAudio running or even installed.

**Comment by PavanLuca** [comment]
Alright, then it's a problem with both PulseAudio and PipeWire or, simply, any audio service.

**Comment by ColinKinloch** [comment]
I'd guess this is a problem in flash player projector itself so unlikely to be fixable beyond patching the binary.
For debugging the issue I'd recommend `pw-top`.   
If you're wanting a good alternative that doesn't have this issue you could use ruffle which doesn't seem to have this issue, at leats in Canabalt.

**Comment by hadess** [comment]
> If you're wanting a good alternative that doesn't have this issue you could use ruffle which doesn't seem to have this issue, at leats in Canabalt.

100%.

Ruffle is available here:
https://flathub.org/apps/rs.ruffle.Ruffle

And if something doesn't work there, it can actually be fixed.
